reference/docs-conceptual/learn/shell/using-light-theme.md

```diff
@@ -0,0 +1,138 @@
+ContinuationPromptColor                : "`e[37m"
+DefaultTokenColor                      : "`e[37m"
+EmphasisColor                          : "`e[96m"
+ErrorColor                             : "`e[91m"
+InlinePredictionColor                  : "`e[38;5;238m"
+KeywordColor                           : "`e[92m"
+ListPredictionColor                    : "`e[33m"
+ListPredictionSelectedColor            : "`e[48;5;238m"
+MemberColor                            : "`e[97m"
+NumberColor                            : "`e[97m"
+OperatorColor                          : "`e[90m"
+ParameterColor                         : "`e[90m"
+SelectionColor                         : "`e[30;47m"
+StringColor                            : "`e[36m"
+TypeColor                              : "`e[37m"
+VariableColor                          : "`e[92m"
+```
+
+The color settings are stored as strings containing ANSI escape sequences that change the color in
+your terminal. Using the `Set-PSReadLineOption` cmdlet you can change the colors to values that work
+better for a light-colored background.
+
+## Defining colors for a light theme
+
+The original PowerShell ISE can be configured to use a light theme for both the editor and console
+panes. You can also view and change the colors that the ISE uses for various syntax and output
+types. You can use these color choices to define a similar theme for **PSReadLine**.
+
+The following hashtable defines colors for **PSReadLine** that mimic the colors in the PowerShell
+ISE.
+
+```powershell
+$ISETheme = @{
+    Command                  = $PSStyle.Foreground.FromRGB(0x0000FF)
+    Comment                  = $PSStyle.Foreground.FromRGB(0x006400)
+    ContinuationPrompt       = $PSStyle.Foreground.FromRGB(0x0000FF)
+    Default                  = $PSStyle.Foreground.FromRGB(0x0000FF)
+    Emphasis                 = $PSStyle.Foreground.FromRGB(0x287BF0)
+    Error                    = $PSStyle.Foreground.FromRGB(0xE50000)
+    InlinePrediction         = $PSStyle.Foreground.FromRGB(0x93A1A1)
+    Keyword                  = $PSStyle.Foreground.FromRGB(0x00008b)
+    ListPrediction           = $PSStyle.Foreground.FromRGB(0x06DE00)
+    Member                   = $PSStyle.Foreground.FromRGB(0x000000)
+    Number                   = $PSStyle.Foreground.FromRGB(0x800080)
+    Operator                 = $PSStyle.Foreground.FromRGB(0x757575)
+    Parameter                = $PSStyle.Foreground.FromRGB(0x000080)
+    String                   = $PSStyle.Foreground.FromRGB(0x8b0000)
+    Type                     = $PSStyle.Foreground.FromRGB(0x008080)
+    Variable                 = $PSStyle.Foreground.FromRGB(0xff4500)
+    ListPredictionSelected   = $PSStyle.Background.FromRGB(0x93A1A1)
+    Selection                = $PSStyle.Background.FromRGB(0x00BFFF)
+}
+```
+
+In PowerShell 7.2 and higher you can use the `FromRGB()` method of `$PSStyle` to create the ANSI
+escape sequences for the colors you want.
```

Comment:
Might be worth turning this into an alert and clarifying in the sentence immediately preceding the code that the hashtable only works on 7.2 and higher - a user might try to copy-paste that into 5.1 if they're not reading carefully.

I don't see an easy way to show it working in 5.1 given it would have to generate the strings inline, so I think just being clear that this is how you can do it in 7.2 and later is good enough.

Reply:
Made it a note